lib/perl/Genome/Model/SomaticValidation/Command/IdentifyDnp.pm

```diff
@@ -55,7 +55,7 @@ sub params_for_result {
     my $build = $self->build;
     my @result_users = Genome::SoftwareResult::User->get(user => $build);
     my @results = map($_->software_result, @result_users);
-    my @merged_results = grep { $_->isa('Genome::InstrumentData::AlignedBamResult') } @results;
+    my @merged_results = grep { ! $_->isa('Genome::InstrumentData::AlignmentResult') } grep { $_->isa('Genome::InstrumentData::AlignedBamResult') } @results;
```

Comment:
Maybe there should be a class that those things inherit from instead of `Genome::InstrumentData::AlignedBamResult`?  I don't know enough about the bigger picture.  The fact that you have had to make the same change in more than one place hints there is something wrong.  I just don't know what the right solution is yet.  Don't consider anything I'm saying as blocking the PR if there's a bug to be fixed.